packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -574,20 +569,27 @@ export const ContentPanel: FC = () => {
                                         resultsTableRunnerByChartType &&
                                         resultsTableRunnerByChartType[
                                             selectedChartType
-                                        ] && (
+                                        ] &&
+                                        (chartVizQuery.data ? (
+                                            <ChartVizTransformedDataTable
+                                                transformedData={
+                                                    chartVizQuery.data
+                                                }
+                                            />
+                                        ) : (
                                             <Table
                                                 resultsRunner={
                                                     resultsTableRunnerByChartType[
                                                         selectedChartType
                                                     ]!
                                                 }
-                                                config={
+                                                columnsConfig={
                                                     tableConfigByChartType[
                                                         selectedChartType
-                                                    ]
+                                                    ]?.columns ?? {}
                                                 }
                                             />
-                                        )}
+                                        ))}
```

Comment:
Since we have `resultsTableRunnerByChartType` available here, should this just be `<Table>`?
Just trying to understand the need for the `ChartVizTransformedDataTable` component